src/pkg/repository/repository.go

```diff
@@ -238,6 +240,27 @@ func (r repository) Backup(ctx context.Context, id model.StableID) (*backup.Back
 	return sw.GetBackup(ctx, id)
 }
 
+// BackupsByID lists backups by ID. Returns as many backups as possible with
+// errors for the backups it was unable to retrieve.
+func (r repository) BackupsByID(ctx context.Context, ids []model.StableID) ([]*backup.Backup, error) {
```

Comment:
Can this func be the canonical `Backups()` and the current func become `BackupsByTag()` or some such?  Or, alternatively, would the `IDs` make an appropriate FilterOption within the current Backups(), instead of a new method?

Reply:
sure, we can call it `Backups()`. I'll pull that into a separate patch when it makes sense. Having it as part of the options seems like it would just make things complicated in the long run, especially since we already have a function to return a backup given an ID